app/src/main/java/com/zackhsi/kiva/activities/LoanReviewActivity.java

```diff
@@ -113,6 +114,7 @@ protected void onActivityResult (int requestCode, int resultCode, Intent data) {
                     String accountId = new KivaProxy().getKivaProxyId();
                     payment.setUserId(accountId);
                     payment.saveEventually();
+                    Toast.makeText(getApplicationContext(), loan.name + " says 'Thanks!'", Toast.LENGTH_LONG).show();
```

Comment:
@zackhsi  is this wrong? I feel like you had to fix something like this previously